src/wsl.star

```diff
@@ -0,0 +1,89 @@
+# System Imports
+system_variables = import_module("github.com/logos-co/wakurtosis/src/system_variables.star")
+
+# Module Imports
+files = import_module(system_variables.FILE_HELPERS_MODULE)
+
+def create_wsl_config():
+    
+    template_data = None
+
+    # Traffic simulation parameters
+    wsl_yml_template = """
```

Comment:
This is what you meant with the "configuration hardcoded" right?

Reply:
Yes, that s right. All the variables in the .yml file that is generated are static at the moment